_dev/Makefile

```diff
@@ -99,18 +99,25 @@ build-all: \
 	build-test-tris-client \
 	build-test-tris-server \
 	build-test-bogo \
+	build-esnitool \
 	$(addprefix build-test-,$(TARGET_TEST_COMPAT))
 
 # Builds TRIS client
 build-test-tris-client: $(BUILD_DIR)/$(OS_ARCH)/.ok_$(VER_OS_ARCH)
-	cd $(DEV_DIR)/tris-testclient; CGO_ENABLED=0 GOROOT="$(GOROOT_LOCAL)" $(GO) build -v -i .
+	cd $(DEV_DIR)/tris-testclient && GOROOT="$(GOROOT_LOCAL)" $(GO) get -v && \
+	   GOOS=linux CGO_ENABLED=0 GOROOT="$(GOROOT_LOCAL)" $(GO) build -v -i .
 	$(DOCKER) build -t tris-testclient $(DEV_DIR)/tris-testclient
 
 # Builds TRIS server
 build-test-tris-server: $(BUILD_DIR)/$(OS_ARCH)/.ok_$(VER_OS_ARCH)
-	cd $(DEV_DIR)/tris-localserver; CGO_ENABLED=0 GOROOT="$(GOROOT_LOCAL)" $(GO) build -v -i .
+	cd $(DEV_DIR)/tris-localserver; GOOS=linux CGO_ENABLED=0 GOROOT="$(GOROOT_LOCAL)" $(GO) build -v -i .
```

Comment:
why?

Reply:
The binary that is built is included in a Docker image. Without this change, I had to run `GOOS=linux make -f _dev/Makefile all` in order to have working Docker images.

I can split this in a separate commit if you prefer that?